appserver/extras/payara-micro/payara-micro-core/src/main/java/fish/payara/micro/impl/PayaraMicroImpl.java

```diff
@@ -1116,6 +1119,12 @@ public PayaraMicroRuntime bootStrap() throws BootstrapException {
         }
     }
 
+    private void configurePolicyFactory() {
+        if (System.getProperty("jakarta.security.jacc.PolicyFactory.provider") == null) {
+            System.setProperty("jakarta.security.jacc.PolicyFactory.provider", "org.glassfish.exousia.modules.def.DefaultPolicyFactory");
+        }
+    }
+
```

Comment:
It might make more sense to set this via the payara-boot.properties file? Rather than hardcoding it like this

Reply:
Totally agree! Recommendation implemented.